modules/Microsoft.EventGrid/subscriptions/.test/common/dependencies.bicep

```diff
@@ -0,0 +1,134 @@
+@description('Optional. The location to deploy to.')
+param location string = resourceGroup().location
+
+@description('Required. The name of the Virtual Network to create.')
+param virtualNetworkName string
+
+@description('Required. The name of the Managed Identity to create.')
+param managedIdentityName string
+
+@description('Required. The name of the Event Grid Topic to create.')
+param eventTopicName string
+
+@description('Required. The name of the Service Bus Namespace to create.')
+param serviceBusName string
+
+resource virtualNetwork 'Microsoft.Network/virtualNetworks@2022-01-01' = {
+    name: virtualNetworkName
+    location: location
+    properties: {
+        addressSpace: {
+            addressPrefixes: [
+                '10.0.0.0/24'
```

Comment:
Extract into variable `var addressPrefix = '10.0.0.0/24'`

Reply:
Removed as not required